components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -14,37 +14,49 @@
+        <chef-toolbar>
+          <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
+            <chef-button id="create-button" primary (click)="openCreateModal()">Create Project</chef-button>
+          </app-authorized>
+        </chef-toolbar>
+        <app-authorized [allOf]="['/iam/v2beta/projects', 'get']">
+          <chef-table>
+            <chef-thead>
+              <chef-tr>
+                <chef-th>Name</chef-th>
+                <chef-th>Type</chef-th>
+                <chef-th class="controls"></chef-th>
+              </chef-tr>
+            </chef-thead>
+            <chef-tbody>
+              <chef-tr *ngFor="let project of sortedProjects$ | async">
+                <chef-td>
+                  <a [routerLink]="['/settings/projects', project.id]">{{ project.name }}</a>
+                </chef-td>
+                <chef-td>{{ project.type | iamType }}</chef-td>
+                <chef-td class="controls">
+                  <app-authorized [allOf]="['/iam/v2beta/projects/{id}', 'delete', project.id]">
+                    <chef-control-menu>
+                      <chef-option (click)="startProjectDelete(project)">Delete Project</chef-option>
+                    </chef-control-menu>
+                  </app-authorized>
+                </chef-td>
+              </chef-tr>
+            </chef-tbody>
+          </chef-table>
         </app-authorized>
-      </chef-toolbar>
-      <app-authorized [allOf]="['/iam/v2beta/projects', 'get']">
-        <chef-table>
-          <chef-thead>
-            <chef-tr>
-              <chef-th>Name</chef-th>
-              <chef-th>Type</chef-th>
-              <chef-th class="controls"></chef-th>
-            </chef-tr>
-          </chef-thead>
-          <chef-tbody>
-            <chef-tr *ngFor="let project of sortedProjects$ | async">
-              <chef-td>
-                <a [routerLink]="['/settings/projects', project.id]">{{ project.name }}</a>
-              </chef-td>
-              <chef-td>{{ project.type | iamType }}</chef-td>
-              <chef-td class="controls">
-                <app-authorized [allOf]="['/iam/v2beta/projects/{id}', 'delete', project.id]">
-                  <chef-control-menu>
-                    <chef-option (click)="startProjectDelete(project)">Delete Project</chef-option>
-                  </chef-control-menu>
-                </app-authorized>
-              </chef-td>
-            </chef-tr>
-          </chef-tbody>
-        </chef-table>
-      </app-authorized>
+      </ng-container>
+      <ng-container *ngIf="(sortedProjects$ | async)?.length === 0">
+        <div class="empty-case-container">
+          <h1>Create the first project to get started!</h1>
+        </div>
+        <div class="empty-case-container">
+          <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
+            <chef-button id="create-button" primary (click)="openCreateModal()">Create Project</chef-button>
```

Comment:
`id` attributes must be unique. Also, the only reason the other `chef-button` has the `id` is for a unit test. So I suspect you have a unit test failure due to this. I would suggest just remove it here unless you plan to add a unit test for this case.